src/main/drivers/pwm_mapping.h

```diff
@@ -36,10 +36,10 @@
 #define MAX_INPUTS  8
 
 #define PWM_TIMER_MHZ 1
-//these three have to be the same because of the ppmAvoidPWMTimerClash functions
-#define ONESHOT_TIMER_MHZ 12
+#define ONESHOT125_TIMER_MHZ 8
+#define ONESHOT42_TIMER_MHZ 24
```

Comment:
Doesn't the 411 on the revo run at 100mhz? it would need to be 10mhz to work in that case.